TYPO3.Media/Documentation/References/Commands.rst

```diff
@@ -22,13 +22,13 @@ commands that may be available, use::
 The following reference was automatically generated from code on 2016-06-07
 
 
-.. _`Media Command Reference: TYPO3.MEDIA`:
+.. _`Media Command Reference: Neos.MEDIA`:
```

Comment:
The references are generated automatically, they should not be edit manually anyways…